src/components/HorizontalCard/HorizontalCard.jsx

```diff
@@ -1,33 +1,44 @@
+import { useCartContext } from "../../context/cart-context";
 import "./HorizontalCard.css";
 
-const HorizontalCard = () => {
+const HorizontalCard = ({ item }) => {
+  const { cartState, cartDispatch } = useCartContext();
   return (
     <div className="card-container-horiz">
       <div className="card-img-box">
-        <img
-          className="card-image"
-          src="https://i.pinimg.com/originals/90/a0/ec/90a0ec57079416bfedb2a2c74377c48d.jpg"
-          alt=""
-        />
+        <img className="card-image" src={item.img} alt="" />
       </div>
       <div className="horizon-content-text">
-        <div className="brand-name">Lacoste</div>
-        <div className="category">Men Slim Fit Tshirt</div>
-        <div className="price">Rs. 999</div>
+        <div className="brand-name">{item.title}</div>
+        <div className="category">{item.description}</div>
+        <div className="price">Rs. {item.discountPrice}</div>
         <div className="price-cut-container">
-          <span className="price-cut">Rs.1999</span>
+          <span className="price-cut">Rs.{item.mrp}</span>
+          <span class="price-offer">({item.discountPercent}% off)</span>
         </div>
+
         <div className="horizon-btn-div">
-          <button className="horizon-btn">
-            <i class="fas fa-plus plus-minus"></i>
-          </button>
-          <span>1</span>
-          <button className="horizon-btn">
+          <button
+            className="horizon-btn"
+            onClick={() => cartDispatch({ type: "DECREASE", payload: item })}
```

Comment:
move this type values in to a constants file

Reply:
yes will surely do that at the end thanks